lib/membrane_rtc_engine/endpoint.ex

```diff
@@ -39,7 +39,11 @@ defmodule Membrane.RTC.Engine.Endpoint do
 
   @spec get_track_id_to_metadata(endpoint :: t()) :: %{Track.id() => any}
   def get_track_id_to_metadata(%__MODULE__{} = endpoint),
-    do: Map.values(endpoint.inbound_tracks) |> Map.new(&{&1.id, &1.metadata})
+    do:
+      endpoint.inbound_tracks
+      |> Map.values()
+      |> Enum.filter(& &1.active?)
+      |> Map.new(&{&1.id, &1.metadata})
```

Comment:
I am not sure what we should do with this. All other functions from this module return all tracks e.g. `get_audio_tracks` or `get_video_tracks`. Maybe a better option would be to remove inactive track? (I know that I wanted to keep it at first)

Reply:
Changing the name of the function to more accurate like "get_active_track_id_to_metadata" will solve this?